substrate/frame/support/procedural/src/benchmark.rs

```diff
@@ -772,20 +764,12 @@ pub fn benchmarks(
 							c
 						);
 
-						let start_pov = #krate::benchmarking::proof_size();
-						let start_extrinsic = #krate::benchmarking::current_time();
-
-						closure_to_benchmark()?;
-
-						let finish_extrinsic = #krate::benchmarking::current_time();
-						let end_pov = #krate::benchmarking::proof_size();
+						let mut recording = #krate::BenchmarkRecording::default();
+						<SelectedBenchmark as #krate::BenchmarkingSetup<#type_use_generics>>::instance(&selected_benchmark, &mut recording, c, verify)?;
```

Comment:
For the V1 syntax you dont really have to change it since we dont care about extending it, but thanks anyway.

Reply:
ah I just rg all I can find. 
Super nit, but this might lead to slightly better benchmark since now dropping memory will happen at the end of the scope, after the recording